### ISSUE TITLE
How did you generate kitti ground truth and why did you use different learning rate.

### ISSUE THREAD

**Issue by ziqi-zhang** [dontLoveBugs/DORN_pytorch#7]
Hi,

Great work!! I have some questions. Firstly, I wonder how did you generate training ground truth on kitti? Can you please provide a link? Secondly I noticed that you set different learning rate for resnet and aspp module. Is there any reason for that? And thirdly can you please tell me the training details for kitti? Such as learning rate and epochs.

Thanks

**Comment by dontLoveBugs** [comment]
About how to generate kitti ground truth, you can refer to https://github.com/mrharicot/monodepth.
You can find the learning rate and training details in the paper.

**Comment by ziqi-zhang** [comment]
Thanks, I figured it out.

**Comment by ziqi-zhang** [comment]
Hi @dontLoveBugs , I think I didn't figure it out. I used kitti depth dataset and tried to train the network, but it is not convergent and the loss does not decrease as well. I set batch size=3 and lr=1e-4 as in the paper. I also normalized the RGB to [0,1] and divide ground truth depth by 100.

Btw, I don't understand how did you set the alpha and beta. In the code, alpha is set to 0.001 for kitti and 0.02 for nyu, beta is set to 80 for both of them. I think 0.001 may be too small for kitti because it is an outdoor dataset and 80 is too big for NYU. I guess this may affect SID?

**Comment by dontLoveBugs** [comment]
For nyu, beta is 10. You can calculate alpha and beta through calculating min depth and max depth in the ground truth.

**Comment by ziqi-zhang** [comment]
But [here](https://github.com/dontLoveBugs/DORN_pytorch/blob/0e982db5ffca4c0c947f3c40baed5a065276e19b/utils.py#L70) the max of depth is 80 when decoding depth.

**Comment by dontLoveBugs** [comment]
Sorry, it's a mistake.

**Comment by ziqi-zhang** [comment]
But I feel that it's still difficult to become convergent and time consuming. For 120K images, bs=3, it will spend 12 hours for 1 epoch. And after several iteraions I didn't see loss to decrease. So I wonder is that true for you?

**Comment by abeyang00** [comment]
@dontLoveBugs @ziqi-zhang  why did you divide the gt by 100? why didnt u normalize gt to [0,1] like RGB input img?
